### PR TITLE
VIH-10162 Use bookings as primary source for host and individual hearing list

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-10162

### Change description ###

Use Bookings API as primary source for hearing list

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.AcceptanceTests/packages.lock.json [hunk 1]
```diff
@@ -287,8 +287,8 @@
       },
       "BookingsApi.Client": {
         "type": "Transitive",
-        "resolved": "1.46.3",
-        "contentHash": "w7YoK/V5QUsWm0l9T9g2iYWE63v3t8M0+G3ii2PKMg6kkOCE743uZQ3DTUzqwaUhApkCNeSlDqcySNGFuJNYBg==",
+        "resolved": "1.44.58-pr-0718-0002",
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb.AcceptanceTests/packages.lock.json:290]: friendly reminder to update this before merge

File: charts/vh-video-web/values.dev.template.yaml [hunk 2]
```diff
@@ -10,4 +10,5 @@ java:
         EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
         EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
         DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+        VHSERVICES__BOOKINGSAPIURL: https://vh-bookings-api-pr-718.dev.platform.hmcts.net/
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:14]: also a pre-merge reminder

**Review by oliver-scott — APPROVED** [review]

**Review by phildebrah — DISMISSED** [review]
Approve

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**